planemo/tool_builder.py

```diff
@@ -19,7 +19,7 @@
 DEFAULT_CWL_VERSION = "v1.0"
 
 
-TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}" python_template_version="3.5">
+TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}+galaxy0" python_template_version="3.5" profile="20.01">
```

Comment:
```suggestion
TOOL_TEMPLATE = """<tool id="{{id}}" name="{{name}}" version="{{version}}+galaxy0" python_template_version="3.5" profile="21.05">
```

I'd use the oldest release currently supported.

Reply:
Is it necessary to modify the build/lib/planemo/tool_builder.py file?

Reply:
Nope, that's not part of the repository.